bee-message/src/payload/transaction/essence.rs

```diff
@@ -201,25 +201,22 @@ impl TransactionEssenceBuilder {
         let mut total: u64 = 0;
 
         for output in self.outputs.iter() {
-            match output {
```

Comment:
Any specific reason for this change ? I'll need to handle dust very soon so I'll have to put it back

Reply:
Clippy warning

Reply:
Alright, I guess I'll bring it back, all good